.github/workflows/lint_prettier.yml

```
@@ -0,0 +1,17 @@
+name: Prettier and Eslint Test
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    name: Prettier ans Eslint Test
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```
use the latest versions of actions wherever possible